coil-gif/src/main/java/coil/decode/GifDecoder.kt

```diff
@@ -34,7 +35,14 @@ class GifDecoder : Decoder {
         options: Options
     ): DecodeResult {
         val drawable = MovieDrawable(
-            movie = source.use { checkNotNull(Movie.decodeStream(it.inputStream())) },
+            movie = if (SDK_INT <= JELLY_BEAN_MR2) {
+                source.use {
+                    val byteArray: ByteArray? = it.readByteArray()
+                    checkNotNull(Movie.decodeByteArray(byteArray, 0, byteArray!!.size)) { " Null response while decoding Gif" }
```

Comment:
`it.readByteArray()` always returns non-null. Also could you remove the error message to match the other branch? Thanks

Reply:
Yup, leftover from testing. Will fix.